### DISCUSSION TITLE
Using sentence Bert (SBert)

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1878] [Q&A]
Quick question: what is the best configuration when using SBert? So, which classifier, query strategy and balance strategy are best to use?

**Comment by Rensvandeschoot** [comment]
For a short animation explaining the different options, see https://www.youtube.com/watch?v=pQ1t60_RF0c
For a longer read, see this blog post: https://asreview.nl/blog/asreview-model-selection-guide/
To learn more about the roles of each component, you might want to read https://www.nature.com/articles/s42256-020-00287-7 or https://www.mdpi.com/2076-3417/14/9/3842. 

For a simulation study testing many different model combinations for 26 datasets, see [Teijema et al. ](https://osf.io/preprints/psyarxiv/2w3rm)

**Comment by RiannevanderFeen** [comment]
@Rensvandeschoot, hi, I have started Phase 3 and am using sBert with the (only?) neural networks option (called: fully connected neural networks 2 hidden layers). However, it never gets done warming up. 

I uploaded the file from Phase 1 and 2, with 290 relevant and 2714 irrelevant (total dataset of 27367) as prior knowledge. I realize this model takes longer... But how much longer? It now just seems to never get done warming up.

I also tried using sBert + random forest and same thing.
Also tried sBert + fully connected neural networks 2 hidden layers with a much! smaller dataset and same thing (doesn't get finished warming up). 

How many of these 'batches' does it need to go over?

<img width="871" alt="image" src="https://github.com/user-attachments/assets/24bae36b-288e-4ecd-a91a-dc85d5e5513c">



Something about: "(AVX2 FMA) (...) rebuild TensorFlow with the appropriate compiler flags."...

Thanks for any suggestions.

**Reply by RiannevanderFeen** [reply]
Ok, so, I turned off 'sleep mode' on my laptop and it took 1 bath of 856 'things' and about 45 minutes. What those 'things' are, I don't know ;-) But it worked and I am able to move forward with Phase 3!
